src/node_jsvmapi.cc

```diff
@@ -186,9 +204,14 @@ namespace v8impl {
                            info.Data().As<v8::External>()->Value());
     napi_func_cb_info cbinfo =
         JsFunctionCallbackInfoFromV8FunctionCallbackInfo(&info);
+    v8::Isolate *isolate = info.GetIsolate();
     cb(
```

Comment:
I think the `lastException` should always be empty at this point before invoking a callback. Would it be helpful to `assert` that?